cmake/projects/Boost/hunter.cmake

```diff
@@ -104,6 +104,17 @@ hunter_add_version(
     3af972569d4b685145442445e51b3fcace342b31
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    Boost
+    VERSION
+    "1.68.0-p1"
+    URL
+    "https://github.com/nxxm/boost/archive/v1.68.0-p1.tar.gz"
```

Comment:
As soon as you merge https://github.com/hunter-packages/boost/pull/5 I can update the URL here.

Reply:
See https://github.com/hunter-packages/boost/releases/tag/v1.68.0-p1